src/System.Diagnostics.Tools/tests/System/Diagnostics/CodeAnalysis/SuppressMessageAttributeTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace System.Diagnostics.CodeAnalysis.Tests
+{
+    public static class SuppressMessageAttributeTests
+    {
+        [Fact]
+        public static void TestConstructorBasic()
```

Comment:
Why are all of the tests static?  This will impede xunit's parallelization of tests in the assembly.